src/components/main-page/Hero.tsx

```diff
@@ -8,18 +9,23 @@ import dots from "@/assets/dots.png";
 export default function Hero() {
   return (
     <div className="flex flex-col gap-4 lg:gap-10 px-4 lg:px-20">
-      <div className="flex flex-col lg:flex-row-reverse items-center justify-center">
+      <Image
+        className="hidden lg:block absolute right-0 top-14 w-full"
+        src={cellDesktopImage}
```

Comment:
Img looks a bit fuzzy on 24 inch monitor, not super bad but noticeable

Reply:
:sob: